src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
 
-                <p>It is important that no modification/rectification work carried out on the vehicle before the appeal
-                    test as this may affect the outcome of the appeal.</p>
-            </div>
+            </ul>
         </div>
+    </div>
+    <!-- second box -->
+    <div class="section-border">
+        <div class="section-title-border">
+            <h1>USE OF THE VEHICLE</h1>
+        </div>
+        <div class="section-text-wrapper">
+            <p id="no-margin-p">It is an offence to use a vehicle that requires type approval without an Individual Approval
+                Certificate confirming compliance with the type approval requirements, except for the purposes of:</p>
+            <ul class="vehicle-section-list">
+                <li> - submitting it, by previous arrangement, for an approval examination;</li>
+                <li> - bringing it away from such an examination;</li>
+                <li> - delivering it, by previous arrangement, to a place where relevant work is to be done on it following
+                    receipt of a refusal to issue an Individual Approval Certificate;</li>
+                <li> - bringing it away from a place where relevant work has been done on it.</li>
+            </ul>
+            <p>You may have been notified of defects in addition to those appertaining to the approval requirements.
+                Even in the above circumstances you may be committing an offence if you use the vehicle, and it does
+                not comply with the various regulations relating to its construction and use.</p>
+        </div>
+    </div>
 
-        <!-- footer -->
-        <div class="footer">
-            <p id="footer-left"> <strong class="bold-title">IVA 30E</strong> (DVSA 0211) </p>
-            <p id="footer-right"> Version 2.3 (30/11/2022) </p>
+    <!-- third box -->
+    <div class="section-border">
+        <div class="section-title-border">
+            <h1>YOUR RIGHT TO APPEAL</h1>
         </div>
+        <div class="section-text-wrapper">
+
+            <p>If you do not agree with the refusal to issue an Individual Approval Certificate, you may appeal. If you
+                wish to appeal against all or any of the grounds for refusal you must complete form IVA17 which is
+                available from the test location or from the gov.uk website <strong>www.gov.uk/vehicleapproval</strong>. The form, together
+                with the appropriate fee, must be received at the Vehicle Approvals Section, DVSA, Ellipse, Padley
+                Road, Swansea, SA1 8AN within 14 days of the date the original notification of refusal was issued.</p>
+
+            <p>It is important that no modification/rectification work carried out on the vehicle before the appeal
+                test as this may affect the outcome of the appeal.</p>
+        </div>
+    </div>
+
+    <!-- footer -->
+    <div class="footer">
```

Comment:
May be one to check, but would think this footer should be at foot of both pages, irrespective of what has been populated.